moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_manipulation.cpp

```diff
@@ -89,9 +89,9 @@ void MotionPlanningFrame::processDetectedObjects()
       object_ids.clear();
       for (const auto& object : *world)
       {
-        if (!object.second->shape_poses_.empty())
+        if (!object.second->global_shape_poses_.empty())
         {
-          const auto& position = object.second->shape_poses_[0].translation();
+          const auto& position = object.second->global_shape_poses_[0].translation();
```

Comment:
I suppose so. I added the change, but I've never used this functionality. I added another note to `MIGRATION.md` although I don't think this will cause issues.